tests/unit/test_ext.py

```diff
@@ -21,6 +27,32 @@ class VendorTornadoTest(TestCase):
     Ensure we are not using any non vendor'ed tornado
     '''
 
+    def test_import_override(self):
+        tmp = tempfile.mkdtemp()
+        test_source = tests.support.helpers.dedent('''
+        from __future__ import absolute_import, print_function
+        import salt
+        import tornado
+        print(tornado.__name__)
```

Comment:
Do we want this print statement?

Reply:
We read it [here](https://github.com/saltstack/salt/pull/56005/files#diff-9ae9edc00b527e581561867b23d4460eR53)